lambda_functions/stac/stac_parent_update.py

```diff
@@ -0,0 +1,299 @@
+"""
+Update parent catalogs based on .yaml files corresponding to datasets uploaded in s3. File lists are
+obtained from s3 inventory lists. Incremental updates can be done by using the 'from-date' option to limit
+the selected dataset yaml files modified by a date later than the specified date. Updated catalog files
+are uploaded to the specified bucket.
+
+The s3 yaml file list is obtained from s3 inventory list unless a file list is provided in the command line.
+"""
+
+from collections import OrderedDict
+from pathlib import Path
+import json
+from parse import parse as pparse
+import boto3
+import click
+from dea.aws import make_s3_client
+from dea.aws.inventory import list_inventory
+from pandas import Timestamp
+
+GLOBAL_CONFIG = {
```

Comment:
This is repeated 3 or 4 times, it needs to be extracted or it's going to become inconsistent.